### ISSUE TITLE
The --gitignore option does not seem to work on directories starting with "/"

### ISSUE THREAD

**Issue by mig-hub** [Old-Man-Programmer/tree#1]
I have noticed this while fiddling around with the Rust programming language. Basically when creating a new project with Cargo, it creates a `.gitignore` automatically and ignores the "target" directory.

The command `tree --gitignore` still shows that "target" directory though.

But in the `.gitignore` it is written "/target" and not "target/" as I would personally do, so I am wondering if this is the reason why. I have seen many ignore files with a slash at the beginning even though I find this confusing. And Git also ignores these so I suppose it makes sense to make this consistent.

**Comment by Old-Man-Programmer** [comment]
I guess after re-reading the gitignore manual, that indeed all paths, even
those starting with a / are relative to the directory of the .gitignore
file.  That is confusing.  I guess I'll add that to the pile of things to
fix.

 - Steve

**Comment by mig-hub** [comment]
Yeah definitely confusing. Once I've seen this I've checked around and was surprised to see how many repos use this.

I am sure you have other things to fix first. I am a bit rusty on C but when I get time I'll see if I can fix this and submit a pull request.
